SCClassLibrary/Common/Core/Nil.sc

```diff
@@ -79,7 +79,12 @@ Nil {
 		// array = array.add(thing);     Instead, it just works.
 		^[value]
 	}
-	addAll { |array| ^array.asArrayCopy }
+	addAll { |array|
+		var result = array.asArray;
+		if(result === array)
```

Comment:
better move the return and adjust the syntax to the usual:
```
if(result === array) { 
		result.copy 
} { 
		result
}
```

Reply:
I've updated the return syntax as you suggested in your first comment.